components/LocationMap.tsx

```diff
@@ -0,0 +1,24 @@
+import { GoogleMap, useLoadScript } from '@react-google-maps/api'
+import { GeoPoint } from 'firebase/firestore'
+import { useMemo } from 'react'
+
+export default function LocationMap({ location }: { location: GeoPoint }) {
+  const center = useMemo(
+    () => ({ lat: location.latitude, lng: location.longitude }),
+    [location.latitude, location.longitude]
+  )
+
+  const { isLoaded } = useLoadScript({
+    googleMapsApiKey: 'AIzaSyBXHLftgJOGUktfa09s3ykfxF17jqB9uEM',
```

Comment:
does the key need to be hidden?

also is the googleMapsApi related to our firebase project? Or is it your email account?

Reply:
1 > Yeah ideally it should come from a Keystore service.
2 > Related to our Firebase project yes, not my email.